src/main/java/appeng/integration/modules/waila/part/StorageMonitorWailaDataProvider.java

```diff
@@ -62,10 +62,11 @@
 			final IAEStack<?> displayed = monitor.getDisplayed();
 			final boolean isLocked = monitor.isLocked();
 
+			// TODO: generalize
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)